packages/ui/components/pages/Fuse/FusePoolsPage/FuseDashNav/CrossFilterButtons.tsx

```diff
@@ -0,0 +1,68 @@
+import { ButtonGroup, Img } from '@chakra-ui/react';
+import { ChainConfig } from '@midas-capital/types';
+import { Dispatch, useMemo } from 'react';
+
+import { CButton } from '@ui/components/shared/Button';
+import { SimpleTooltip } from '@ui/components/shared/SimpleTooltip';
+import { useMultiMidas } from '@ui/context/MultiMidasContext';
+import { getChainConfig } from '@ui/utils/networkData';
+
+export const CrossFilterButtons = ({
```

Comment:
Let's call it `ChainFilterButtons`

Reply:
Okay, maybe it needs to be refactored by `react-table`